src/main/java/com/puppetlabs/jenkins/plugins/puppetgatling/gatling/PuppetGatlingBuildAction.java

```diff
@@ -67,7 +67,15 @@ public ReportRenderer getReport(String simulationName) {
 	}
 
 	public String getReportURL(String simulationName) {
-		return new StringBuilder().append(URL_NAME).append("/report/").append(simulationName).toString();
+		return new StringBuilder().append("job/").
```

Comment:
I'm not sure how this is consumed exactly but based on the behavior I had been seeing, I was expecting to just have to trim off the `URL_NAME` bit - assuming that expands to `puppet-gatling` since the URL I was getting in the browser with the previous plugin was `/job/<job_name>/<job_instance>/puppet-gatling/puppet-gatling/report/<sim-name>` but the actual working URL would just have one less `puppet_gatling` in it: `/job/<job_name>/<job_instance>/puppet-gatling/report/<sim-name>`.  Does the following not work?

```java
return new StringBuilder().append("/report/").append(simulationName).toString();
```

In any event, I imagine this worked for you in your testing, so happy to merge unless you have time to show me how to build / test this out.